### PR TITLE
add msgpack bulk import example with introducing util.create_msgpack()

### PR DESCRIPTION
**Pull request by chezou**

### CODE REVIEW

File: tdclient/api.py [hunk 1]
```diff
@@ -638,6 +638,14 @@ def _read_tsv_file(self, file_like, **kwargs):
 
 
 def normalized_msgpack(value):
```
Comment (takuti) [inline tdclient/api.py]: Do you think if it's a good idea to move `normalized_msgpack` function to `tdclient.util`?

File: README.rst [hunk 2]
```diff
@@ -131,33 +136,69 @@ Importing data into Treasure Data in batch manner.
...
-           print("imported %d records." % (bulk_import.valid_records,))
+           print("imported {} records.".format(bulk_import.valid_records))
        else:
-           raise(RuntimeError("no records have been imported: %s" % (repr(bulk_import.name),)))
+           raise(RuntimeError("no records have been imported: {}".format(repr(bulk_import.name))))
        bulk_import.commit(wait=True)
        bulk_import.delete()
 
+
+If you want to import data as `msgpack <https://msgpack.org/>`_ format, you can write as follows:
```
Comment (takuti) [inline README.rst]: Nice addition, but it's a bit hard to see diff from the above example. Don't you simply drop the latter lines?:

```py
import io
import time
import uuid
import warning

import tdclient

t1 = int(time.time())
l1 = [{"a": 1, "b": 2, "time": t1}, {"a": 3, "b": 9, "time": t1}]

with tdclient.Client() as td:
    session_name = "session-{}".format(uuid.uuid1())
    bulk_import = td.create_bulk_import(session_name, "mydb", "mytbl")
    try:
        _bytes = tdclient.util.create_msgpack(l1)
        bulk_import.upload_file("part", "msgpack", io.BytesIO(_bytes))
        bulk_import.freeze()
    except:
        bulk_import.delete()
        raise
    bulk_import.perform(wait=True)
    # same as the above example
```

File: tdclient/api.py (hunk 1 above)
Comment (chezou) [reply tdclient/api.py]: agreed 6fd3a6f

File: README.rst (hunk 2 above)
Comment (chezou) [reply README.rst]: sounds good 6fd3a6f

File: tdclient/api.py (hunk 1 above)
Comment (chezou) [reply tdclient/api.py]: Note: Need to be fixed after releasing new tdclient https://github.com/treasure-data/pytd/blob/master/pytd/writer.py#L11

File: README.rst [hunk 3]
```diff
@@ -131,33 +136,62 @@ Importing data into Treasure Data in batch manner.
...
        bulk_import.perform(wait=True)
        if 0 < bulk_import.error_records:
-           warnings.warn("detected %d error records." % (bulk_import.error_records,))
+           warnings.warn("detected {} error records.".format(bulk_import.error_records))
        if 0 < bulk_import.valid_records:
-           print("imported %d records." % (bulk_import.valid_records,))
+           print("imported {} records.".format(bulk_import.valid_records))
        else:
-           raise(RuntimeError("no records have been imported: %s" % (repr(bulk_import.name),)))
+           raise(RuntimeError("no records have been imported: {}".format(repr(bulk_import.name))))
```
Comment (takuti) [inline README.rst]: Drop `repr`

Comment (chezou) [reply README.rst]: Thanks 6fab330

**Pull request merged**